### DISCUSSION TITLE
23.04 support?

### DISCUSSION THREAD

**Discussion by User8395** [waydroid/waydroid#877] [Q&A] [answered]
My system said it has to remove Waydroid to install 23.04.

Is this normal?

**Comment by cniw** [comment, marked as answer]
yes it's normal because waydroid packages still host on our (3rd-party) repository,
you'll see this similar behavior again when do updating ubuntu/debian version because it's ship different python package version,
until in future waydroid packages shiped in official ubuntu/debian repository

see https://github.com/waydroid/waydroid/issues/858 to solve your issue
